packages/client/lib/commands/MIGRATE.ts

```diff
@@ -9,59 +10,56 @@ export interface MigrateOptions {
 
 export default {
   IS_READ_ONLY: false,
-  transformArguments(
+  parseCommand(
+    parser: CommandParser,
     host: RedisArgument,
     port: number,
     key: RedisArgument | Array<RedisArgument>,
     destinationDb: number,
     timeout: number,
     options?: MigrateOptions
   ) {
-    const args = ['MIGRATE', host, port.toString()],
-      isKeyArray = Array.isArray(key);
+    parser.push('MIGRATE', host, port.toString());
+    const isKeyArray = Array.isArray(key);
   
     if (isKeyArray) {
-      args.push('');
+      parser.push('');
     } else {
-      args.push(key);
+      parser.push(key);
```

Comment:
change to pushKey()

Reply:
didn't specify key index before, so makes sense why i didn't use pushKeys, but perhaps it should?  or perhaps should never be called from a cluster client?

Reply:
same comment, as above, perhaps makes sense why not.